src/main/proto/phenopackets/schema/v2/core/interpretation.proto

```diff
@@ -58,7 +58,7 @@ message GenomicInterpretation {
     // Gene or variant identified in subject or biosample and interpreted in relation to the disease diagnosis. In this
     // context athe 'gene' is a proxy for an unspecified alteration in the indicated gene.
     oneof call {
-        Gene gene = 3;
+        org.ga4gh.vrsatile.v1.GeneDescriptor gene = 3;
         VariantInterpretation variant_interpretation = 4;
 //        VrsatileInterpretation vrsatile_interpretation = 5;
```

Comment:
Can  you add this message in, please?

Reply:
Add the Gene message in addition to the GeneDescriptor? Our hope is that we can have a single representation of Gene using the GeneDescriptor from VRSATILE. It would be confusing to allow both. Or am I misunderstanding this request and you're saying that the `org.ga4gh.vrsatile.v1.GeneDescriptor` message isn't showing up?